### DISCUSSION TITLE
Remove methods which duplicates .NET MAUI methods

### DISCUSSION THREAD

**Discussion by VladislavAntonyuk** [CommunityToolkit/Maui#377] [General]
Analyze and remove methods that already exist in .NET MAUI. Example:
![image](https://user-images.githubusercontent.com/33021114/164551543-f0327fb0-39d2-4b72-adbd-ab5e6295a4cd.png)

It would be nice to finalize it before GA.

**Comment by TheCodeTraveler** [comment]
Agreed. The only thought I have is should we do it after GA since our RC contains these APIs?

**Reply by VladislavAntonyuk** [reply]
I suggest to remove it now. In that case we affect less customers. Or we can mark them as obsolete in RC2. And remove in GA.

**Reply by bijington** [reply]
Obsolete would get my vote as the first step

**Reply by VladislavAntonyuk** [reply]
I am just thinking that releasing a new product with the Obsolete attribute looks weird

**Reply by bijington** [reply]
Yeah that is a very good point, shipping a v1 with something Obsolete would feel wrong

**Comment by CliffAgius** [comment]
Closed as complete
